### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by prembala1710**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# CliqInformerProject
+This Project is to Deploy Github Workflows
```
Comment (prem-bkp-test-github) [inline README.md:2]: Add more Data.

**Review by prem-bkp-test-github — CHANGES_REQUESTED** [review]
Start Working on This @prembala1710

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
```
Comment (prembala1710) [inline README.md:2]: This Line is Not Required.

**Comment by prembala1710** [comment]
This is Comment

**Pull request closed without merging**